package.json

```diff
@@ -27,7 +27,9 @@
     "commitizen": "2.9.6",
     "semantic-release": "^6.3.2"
   },
-  "czConfig": {
-    "path": "./"
+  "config": {
+    "commitizen": {
+      "path": "./index.js"
```

Comment:
the filename should be included here,
otherwise commitizen will throw errors.
dont know why 🤔